packages/common/src/Config.ts

```diff
@@ -103,7 +103,7 @@ const getEnvVar = (envVar: string): string => {
   }
 }
 
-export const appConfig: CCFConfig = {
+const getConfig = (): CCFConfig => ({
```

Comment:
Looks good, but this breaks out test coverage thresholds, but y'all don't need to worry about that. Our team can resolve any coverage issues.

Reply:
Sorry, we can write a test, that's not a problem.